CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Possible log types:
 
 ### Unreleased
 
+- [changed] Update to Rust 2018 edition (#76)
```

Comment:
```suggestion
- [changed] Require Rust 1.31+ (Rust 2018 edition) (#76)
```

Reply:
Hmm. But that's not really what has changed. On the other hand it's what the user probably interests.

Reply:
Yes, the user doesn't care about Rust 2018, but about the fact that it won't build with Rust <1.31 anymore.